pipeline/load/display_metrics.py

```diff
@@ -34,5 +62,18 @@ def display_internal_matching_metrics(
         output.users, metrics.n_proposed_matches_per_user
     )
     matched_user_emails = render_user_emails(output.users)
+    counts_per_intent_in_matches = render_intent_distribution_in_matches(
+        metrics.distribution_in_final_set
```

Comment:
Good job integrating with the existing MatchingMetrics module to store the metrics!